packages/core/src/components/plekinfo-card/plekinfo-card.scss

```diff
@@ -106,6 +112,18 @@ del.dso-plekinfo-card-container {
   --_dt-rich-content-margin-block-start: 0;
 }
 
+::slotted([slot="interactions"]),
```

Comment:
`::slotted([slot="interactions"])` hoort bij Card en zal in deze stylesheet niets doen.

Reply:
verwijderd